src/app_charts/base/robot/fluent-bit.yaml

```diff
@@ -28,16 +28,37 @@ metadata:
     app.kubernetes.io/managed-by: Helm
 data:
   custom_parsers.conf: |
+    # Merges multi-line Abseil logs. The regexs assume that any line which does not start
```

Comment:
Separate PR?

Reply:
I can put this in a separate PR, I wasn't sure if I was supposed to run the shell script in this PR or not (`fluent-bit-helm.sh`)?

https://github.com/googlecloudrobotics/core/commit/06e09bff2e1e3a45a61052f9e4de3c3747fcb62c

Reply:
I removed the commit that ran the script, and rebased this PR